pp/integrations/plugin-packs/procedures/operatingsystems-mac-snmp.md

```diff
@@ -2,48 +2,853 @@
 id: operatingsystems-mac-snmp
 title: Mac SNMP
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Pack assets
+
+### Templates
+
+The Monitoring Connector **Mac SNMP** brings a host template:
+
+* **OS-Mac-SNMP-custom**
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="OS-Mac-SNMP-custom" label="OS-Mac-SNMP-custom">
+
+| Service Alias | Service Template           | Service Description                                                                                                                                |
+|:--------------|:---------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------|
+| Cpu           | OS-Mac-Cpu-SNMP-custom     | Check the rate of utilization of CPU for the machine. This check can give the average CPU utilization rate and the rate per CPU for multi-core CPU |
+| Disk-IO       | OS-Mac-Disk-IO-SNMP-custom | Check access disk of the disk. For each check the name of the disk will appear "label" rather than the letter assigned                             |
```

Comment:
```suggestion
| Disk-IO       | OS-Mac-Disk-IO-SNMP-custom | Check disk access. For each check the name of the disk will appear as a "label" rather than the letter assigned                             |
```

Reply:
as a label?